src/main/java/com/ostapiuk/core/providers/DataObjectsProvider.java

```diff
@@ -0,0 +1,37 @@
+package com.ostapiuk.core.providers;
+
+import com.ostapiuk.business.model.UserEntity;
+import com.ostapiuk.core.utils.JsonParser;
+import org.testng.annotations.DataProvider;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+public class DataObjectsProvider {
```

Comment:
Name!!!!
If it is data provider for users, please give a proper name:
UsersDataProvider